docs/content/guides/rows/row-sorting.md

```diff
@@ -1,79 +1,141 @@
 ---
 id: 6o0zftmc
-title: Row sorting
-metaTitle: Row sorting - JavaScript Data Grid | Handsontable
-description: Sort your data in ascending, descending, or a custom order, across one or multiple rows of the data grid.
+title: Rows sorting
+metaTitle: Rows sorting - JavaScript Data Grid | Handsontable
+description: Sort rows alphabetically or numerically, in ascending, descending or a custom order, across one or multiple columns.
 permalink: /row-sorting
 canonicalUrl: /row-sorting
+tags:
```

Comment:
Cool, but I would add some of the following:

- row ordering
- order by
- sort ascending
- sort descending
- highest to lowest
- unordered data
- sortEmptyCells

I would remove
- columns sorting (repetition)